CHANGELOG.md

```diff
@@ -5,7 +5,9 @@ Multitenant AWS Dynamo supports the [AWS Dynamo Java API](http://docs.aws.amazon
   
 You can write your application code against the Amazon DynamoDB interface as you would for any other application.  The implementation will manage storage of data by tenant.
 
-## 0.9.19 (unreleased)
+## 0.9.20 (unreleased)
+
+* Fixed `UnsupportedOperationException` exception when performing an update request on a GSI hash key attribute
```

Comment:
doesn't this conflict with PR #168 ?

Reply:
Github will probably flag it as a conflict, but I will manually resolve it to two different entries as they are independent bugs.